posts/2021/05/may-status-update.md

```diff
@@ -0,0 +1,101 @@
+<!--
+.. title: HPy Status Update
+.. slug: may-status-update
+.. date: 2021-05-07 10:00:00 UTC
+.. author: hodgestar
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+
+HPy's first git commit was on Friday, 12 July 2019. Despite the crazy state
+of the world, HPy has come a long way since then. This is a quick update
+to say where we are now and what's planned for the immediate future.
+
+<!--TEASER_END-->
+
+What can one do with HPy right now?
+===================================
+
+One can already port non-trivial Python extensions to the HPy API, but
+there is also still a lot of work to do. Things one can do right now
+with HPy include:
+
+  - Initialize an extension module
+  - Create custom extension types
+  - Create custom methods and functions
+  - Interact with many built-in types (including dictionaries, tuples,
+    lists, strings, bytes and longs)
+  - Raise and check exceptions
+  - Parse arguments to C functions
+  - Call Python functions
+  - Compile an extension in either "universal" or "cpython" modes (see below)
+  - Put a "universal" extension into debug mode **at runtime** (see below
+    for how this is possible without a performance penalty).
+  - Compile extensions for POSIX and Windows (MSC).
+
+An extension written using HPy may be compiled either in "cpython" or
+"universal" mode:
+
+  - "cpython" mode produces an ordinary Python C extension that only uses
+    the HPy API internally. From the point of view of the Python implementation
+    it is an ordinary Python C extension.
```

Comment:
```suggestion
  - "cpython" mode produces a c-extension that is fully compatible with current
    C-API c-extensions and is loaded via the standard import machinery.
```

Reply:
Your version is more technically precise, but I was aiming for something that readers of the blog post who aren't necessarily dealing with Python internals day to day would be able to read easily and gain the correct overall picture from.